timetracker/timetracker/urls.py

```diff
@@ -19,7 +19,20 @@
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	name = request.GET.get("name", None)
```

Comment:
`request.GET.get("name")` will return `None` by default if "name" doesn't exist, but being explicit is okay.